src/config/config.ts

```diff
@@ -287,21 +288,44 @@ export class Config implements IConfig {
     .toUpperCase()
   }
 
-  findCommand(id: string, opts: {must: true}): Command.Plugin
-
-  findCommand(id: string, opts?: {must: boolean}): Command.Plugin | undefined
-
-  findCommand(id: string, opts: {must?: boolean} = {}): Command.Plugin | undefined {
-    const command = this.commands.find(c => c.id === id || c.aliases.includes(id))
-    if (command) return command
-    if (opts.must) error(`command ${id} not found`)
+  findCommand(id: string, opts: { must: true }): Command.Plugin
+
+  findCommand(id: string, opts?: { must: boolean }): Command.Plugin | undefined
+
+  findCommand(id: string, opts: { must?: boolean } = {}): Command.Plugin | undefined {
```

Comment:
Modified to defer to core when the duplicates are from core and a user installed (install or link) plugin.

Reply:
Can you add a comment here summarizing the preference order and why we want to check for that